pkg/apiserver/domain/service/application_test.go

```diff
@@ -936,35 +936,37 @@ var _ = Describe("Test apiserver policy rest api", func() {
 		checkRes, err = json.Marshal(checkWorkflow.Steps[0].Properties)
 		Expect(err).Should(BeNil())
 		Expect(string(checkRes)).Should(BeEquivalentTo(`{"policies":["cluster3","override1"]}`))
+
+		policyRes, err = appService.DetailPolicy(ctx, appModel, policyName)
+		Expect(err).Should(BeNil())
+		Expect(policyRes.WorkflowPolicyBindings).Should(BeEquivalentTo([]v1.WorkflowPolicyBinding{
+			{
+				Name:  "second",
+				Steps: []string{"second"},
+			},
+		}))
 	})
 
-	It("Exsit binding will block policy delete operation", func() {
+	It("Exist binding will block policy delete operation", func() {
 		appModel, err := appService.GetApplication(context.TODO(), testApp)
 		Expect(err).Should(BeNil())
 		policyName := "override1"
 		_, err = appService.DetailPolicy(ctx, appModel, policyName)
 		Expect(err).Should(BeNil())
-		err = appService.DeletePolicy(ctx, appModel, policyName)
+		err = appService.DeletePolicy(ctx, appModel, policyName, false)
 		Expect(err).ShouldNot(BeNil())
 	})
 
-	It("Update workflow delete using step will unblock delete", func() {
+	It("Force delete policy will delete policy workflow binding", func() {
 		appModel, err := appService.GetApplication(context.TODO(), testApp)
 		Expect(err).Should(BeNil())
 		policyName := "override1"
-		policyRes, err := appService.DetailPolicy(ctx, appModel, policyName)
-		Expect(err).Should(BeNil())
-		propertyStr, err := json.Marshal(policyRes.Properties)
-		Expect(err).Should(BeNil())
-		updatePolicyReq := v1.UpdatePolicyRequest{
```

Comment:
Why remove the test case about the update.

Reply:
Force delete will do such things.